### DISCUSSION TITLE
lite weight browser K-Meleon 76.4.6

### DISCUSSION THREAD

**Discussion by nydave69** [pymedusa/Medusa#10808] [General]
I am trying to use K-Meleon for my APP server that runs Medusa and a few other apps....
All the others work fine..    I just get a blank page when try to connect to Medusa...
I had this problem with chrome and found that the Ad blocker was stopping the login popup...
But K-Meleon 76.4.6 doesn't have a Ad blocker, and it's popup blocker is disabled..

Any guesses or help...     
Would really like to use it since it's small and doesn't use many resources...

**Comment by p0psicles** [comment]
Afaik Medusa doesn't have a login popup?
But my time is limited. So that's why we decided to limit browser support to latest two major versions of ff, edge and chrome.

**Comment by p0psicles** [comment]
But maybe someone else is okay with spending time on this

**Comment by nydave69** [comment]
Thanks...

I'll look into it some more, if I find an answer I'll post it...
